src/dotnet-interactive.Tests/StdIoKernelConnectorTests.cs

```diff
@@ -120,11 +120,38 @@ public async Task it_can_create_a_proxy_kernel_to_more_than_one_remote_subkernel
                 SupportedKernelCommands = fsharpKernelInfo.SupportedKernelCommands
             };
 
+            const string installHttpKernel =
+                """
+                #r "Microsoft.DotNet.Interactive.HttpRequest.dll"
```

Comment:
This is unlikely to behave the way we expect it to in the context of the test project. We should discuss.

Reply:
This does seem to work (including in CI) :) But will discuss offline.